src/components/container/LabResultsBloodType/LabResultsBloodType.tsx

```diff
@@ -0,0 +1,165 @@
+import MyDataHelps from "@careevolution/mydatahelps-js";
+import React, { useEffect, useState } from "react";
+import { Action, LoadingIndicator } from "../../presentational";
+import "./LabResultsBloodType.css";
+import "../HealthPreviewSection/HealthPreviewSection.css"
+import language from "../../../helpers/language";
+import { bloodTypeLabs, singleLabs, manyLabs } from "./LabResultsBloodType.previewdata";
+import { faInfoCircle } from "@fortawesome/free-solid-svg-icons";
+import iconBloodType from "./icon-bloodtype.svg";
+
+export interface LabResultsBloodTypeProps {
+    previewState?: "BloodTypeLabs" | "SingleLabs" | "ManyLabs" | "NoData"
+    summaryResultsOnly?: true | false
+    maximumResults?: number
+    onClick?: any
+    externalInfoLink?: true | false
```

Comment:
This one is never used. Can be removed

Reply:
Not sure the best way to hand displaying the info site -- looking into this

Reply:
I would remove the externalInfoLink, externalInfoLinkDefault,  externalInfoLinkSpanish props.

I think if onClick is supplied we should use that like you have it - but otherwise just use hardcoded medline plus links so there is sensible default behavior (basically the user would override onClick if they want to override where the component goes)

Don't think I see a strong use case for suppressing the info link entirely right now so we can cross that bridge later if we come to it

Reply:
Updated to remove all of these parameters

Reply:
Fixed